### PR TITLE
Document Parquet and ORC compression support

### PR DESCRIPTION
**Pull request by sameerz**
1. Please write a description in this text box of the changes that are being
   made.

Add documentation on compression formats supported for Parquet and ORC on read and write.  Closes issue #341 

2. Please ensure that you have written units tests for the changes made/features
   added.

Reviewed in .md file in github.

### CODE REVIEW

File: docs/compatibility.md [hunk 1]
```diff
@@ -184,6 +186,8 @@ Parquet will not be GPU-accelerated. If the INT96 timestamp format is not requir
 compatibility with other tools then set `spark.sql.parquet.outputTimestampType` to
 `TIMESTAMP_MICROS`.
 
+The plugin supports reading `uncompressed`, `snappy` and `gzip` Parquet files and writing `uncompressed` and `snappy` Parquet files.
```
Comment (revans2) [inline docs/compatibility.md:189]: I think the important thing we are missing is that we cannot tell what the file is compressed with until we start to read it and at that point we don't have the ability to fall back to the CPU so you will get an error. This is also true for ORC.

Comment (sameerz) [reply docs/compatibility.md]: Updated to note that we will error out on reading or writing an unsupported format

File: docs/compatibility.md [hunk 2]
```diff
@@ -164,6 +164,10 @@ similar issue exists for writing dates as described
 appears to work for dates after the epoch as described
 [here](https://github.com/NVIDIA/spark-rapids/issues/140). 
 
+The plugin supports reading `uncompressed`, `snappy` and `zlib` ORC files and writing `uncompressed`
+ and `snappy` ORC files.  At this point, the plugin does not have the ability to fall back to the 
+ CPU when reading or writing an unsupported compression format, and will error out in that case.
```
Comment (revans2) [inline docs/compatibility.md:169]: Sorry it is just reading.  Writing we will fall back to the CPU for an unsupported format.

Comment (sameerz) [reply docs/compatibility.md]: Corrected to only indicate reading will error out.

**Comment by sameerz** [comment]
build

**Pull request merged**